docs/signingin.md

```diff
@@ -0,0 +1,44 @@
+# Signing in and signing up
+
+## Introduction
+
+Signing in means you click a link to go straight to a Kuma endpoint which
+will redirect you back to where you were.
+
+Yari produces a SPA app at `/$locale/signin`. It presents to clickable
+links to our supported identity providers. Clicking the "Sign in" in the
+header will note which page you were on, then it goes to
+`/$locale/signin?next=page/you/were/on`
+
+When the user picks a identity provider they leave the static site
+nature of Yari and dynamically render a Kuma page which itself,
+will trigger a redirect to the identity providers authentication screen.
+If the user completes that, it will redirect back to Kuma which then
+makes a decision...
+
+1. If you've signed in before (aka. signed up) you're redirected back to
+   that original `?next=...` variable that was set when you clicked "Sign in".
```

Comment:
Since I couldn't find the respective redirect code in this review view:
Do we check, whether the value is valid (as in: points to a known path instead of evildomain.tld and the path exists)?

Otherwise I'd be worried, that it could be to hijack users to a malware page.

Reply:
Fully taken care of my Kuma. Well tested.